orttraining/orttraining/training_ops/cuda/controlflow/wait.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <thread>
```

Comment:
seems there are few includes not necessary

Reply:
Removed. Thanks.